### ISSUE TITLE
Scrambled graphics on some layers with v0.7.5

### ISSUE THREAD

**Issue by EvilRyuSp** [Mazamars312/Analogue_Pocket_Neogeo#36]
![image](https://user-images.githubusercontent.com/118354191/202163048-ca27beac-80d8-4840-a991-88f126a53e42.png)
![image](https://user-images.githubusercontent.com/118354191/202163094-6673c9e8-7832-455f-848b-9b57e9676be5.png)

With version 0.7.5 I see lots of scrambled graphics in some layers. Please see screenshots for details (The game used for the example is Andro Dunos).

This happens with all games, multiple bios files, different compilations of the core, version 0.7, different versions of the games, etc. I have reported this on Reddit and got one response of a user having the same issues. My Analogue Pocket is one of the newer ones for Group B preorders (maybe slight differences in memory tolerances in some of the new models?)

My files have been tested on a second Analogue Pocket that did not have any issues to confirm that the configuration was correct.

**Comment by terminator2k2** [comment]
problem seemed to have been solved with a recompile

**Comment by jere08200** [comment]
Excuse me but I have the same problem ! And I don't understand your answer. What's SEED 1 ? How to solve this problem ?

**Comment by terminator2k2** [comment]
> Excuse me but I have the same problem ! And I don't understand your answer. What's SEED 1 ? How to solve this problem ?

go here, and download the core:

https://www.reddit.com/r/AnaloguePocket/comments/ywfgqg/comment/iy6iyr3/?context=3

**Comment by jere08200** [comment]
thank you very much i will try that

**Comment by jere08200** [comment]
excuse me but where should I put the core? in the neo geo alpha 0.7.5 common file?

**Comment by terminator2k2** [comment]
in Cores\Mazamars312.NeoGeo

**Comment by jere08200** [comment]
Thanks a lot ! it works now

**Comment by dji85** [comment]
I got some scrambled sprites on World Heroes 2

**Comment by evanphx** [comment]
Sounds like it's something up with a newer patch of analogue pockets and their memory timing. The rom posted at the link before isn't available anymore, anyone still have it to share?

**Comment by terminator2k2** [comment]
> Sounds like it's something up with a newer patch of analogue pockets and their memory timing. The rom posted at the link before isn't available anymore, anyone still have it to share?

here you go:

https://easyupload.io/ijx3t3

**Comment by evanphx** [comment]
@terminator2k2 Thank you!

**Comment by Mazamars312** [comment]
Most of this stuff is fixed up in the new build. Should be released soon. The only game I know in the new build that has a issue is The Super Spy. which ill look at later

**Comment by Mazamars312** [comment]
This should be resolved in version 0.8.1. Can you please confirm?

**Comment by EvilRyuSp** [comment]
I updated to 0.8.0 yesterday and did not encounter this issue, looks good to me!
